src/main/java/com/tysng/duke/service/Duke.java

```diff
@@ -60,6 +66,12 @@ public Response handle(Command command) {
         return response;
     }
```

Comment:
I haven't encountered anyone else who used streams, so it's nifty. I might consider adding streams in my current Duke as well. Switch cases are appropriate.